app/lib/gateway/optics.rb

```diff
@@ -0,0 +1,41 @@
+module Gateway
+  class Optics
+    class ClientError < StandardError; end
+
+    GET_TOKEN_ENDPOINT = 'https://uat.icasework.com/token?db=hmcts'.freeze
+    ENDPOINT = 'https://uat.icasework.com/createcase?db=hmcts'.freeze
+
+    def request_bearer_token(jwt_token:)
+      rep = post_jwt(jwt_token)
+      json = JSON.parse rep.body, symbolize_names: true
+      json.fetch(:access_token)
+    end
+
+    def post(body:, bearer_token:)
```

Comment:
we have a method called `post_jwt` and `post` recommend we rename post to make clear what this methods does (e.g.`post_case`)

Reply:
Not sure we should make this gateway specific to just cases.
This gateway is just a wrapper around the Optics API itself